res/values/strings.xml

```diff
@@ -183,133 +183,183 @@ there are any).</string>
+    <string name="np_name_invg">Ingolstadt</string>
     <string name="np_desc_invg">Ingolstadt</string>
-    <string name="np_desc_vgn">Nuremberg, Fürth, Erlangen</string>
-    <string name="np_desc_vvm">Swabia, Mittelschwaben, Krumbach, Günzburg, Memmingen</string>
-    <string name="np_desc_vmv">Mecklenburg-Vorpommern, Schwerin</string>
+    <string name="np_desc_kvv">Baden-Württemberg, Karlsruhe</string>
+    <string name="np_desc_mvg">Märkischer Kreis, Lüdenscheid</string>
+    <string name="np_desc_mvv">Bavaria, Munich</string>
+    <string name="np_name_nasa" translatable="false">INSA</string>
+    <string name="np_desc_nasa">Saxony, Leipzig, Sachsen-Anhalt, Magdeburg, Halle</string>
+    <string name="np_desc_nvbw">Baden-Württemberg, Konstanz, Basel, Basel-Stadt, Reutlingen, Rottweil, Tübingen, Sigmaringen</string>
+    <string name="np_name_nvv" translatable="false">NVV/RMV</string>
+    <string name="np_desc_nvv">Hesse, Frankfurt am Main, Kassel</string>
     <string name="np_desc_rsag" tools:keep="@string/np_desc_rsag">Rostock</string>
-    <string name="np_desc_hvv" tools:keep="@string/np_desc_hvv">Hamburg</string>
     <string name="np_desc_sh">Schleswig-Holstein, Kiel, Lübeck, Hamburg</string>
-    <string name="np_desc_gvh">Lower Saxony, Hannover, Hamburg</string>
-    <string name="np_desc_bsvag">Braunschweig, Wolfsburg</string>
+    <string name="np_name_vagfr" translatable="false">VAG</string>
+    <string name="np_desc_vagfr">Elsass, Bas-Rhin, Straßburg, Freiburg im Breisgau</string>
+    <string name="np_desc_vbb">Brandenburg, Berlin</string>
     <string name="np_desc_vbn" tools:keep="@string/np_desc_vbn">Lower Saxony, Bremen, Bremerhaven, Oldenburg (Oldenburg), Osnabrück</string>
-    <string name="np_desc_vvo">Saxony, Dresden</string>
+    <string name="np_desc_vgn">Nuremberg, Fürth, Erlangen, Bayreuth, Bamberg, Schwabach</string>
+    <string name="np_desc_vgn_networks" translatable="false">VAG, DB, ESTW, infra fürth, Stadtwerke Bayreuth, Stadtwerke Schwabach, STWB, GPV, GkV</string>
+    <string name="np_desc_vgs">Saarland, Saarbrücken</string>
     <string name="np_desc_vms">Mittelsachsen, Chemnitz</string>
-    <string name="np_desc_nasa">Saxony, Leipzig, Sachsen-Anhalt, Magdeburg, Halle</string>
-    <string name="np_desc_vrr">North Rhine-Westphalia, Köln, Bonn, Essen, Dortmund, Düsseldorf, Münster, Paderborn, Höxter</string>
-    <string name="np_desc_mvg">Märkischer Kreis, Lüdenscheid</string>
-    <string name="np_desc_nvv">Hesse, Frankfurt am Main, Kassel</string>
+    <string name="np_desc_vmv">Mecklenburg-Vorpommern, Schwerin</string>
     <string name="np_desc_vrn">Baden-Württemberg, Rheinland-Pfalz, Mannheim, Mainz, Trier</string>
+    <string name="np_desc_vrr">North Rhine-Westphalia, Köln, Bonn, Essen, Dortmund, Düsseldorf, Münster, Paderborn, Höxter</string>
+    <string name="np_desc_vrs">Cologne, Bonn</string>
+    <string name="np_name_vvm" translatable="false">move</string>
```

Comment:
So this is not http://vvm-info.de/home/startseite/startseite.jsp since the provider is called VVM internally?